src/reducers/vms.js

```diff
@@ -63,23 +73,14 @@ const vms = actionReducer(initialState, {
     }
     return state
   },
-  [SET_VM_NICS] (state, { payload: { vmId, nics } }) {
```

Comment:
Actually, we want to use this constants in reducers, for saving consistency.

Reply:
True.  I'll follow up this PR with a refactor PR to fix those in all reducers.